### PR TITLE
fix: refactor how the http alerts variables are declared in service.yaml

### PR DESCRIPTION
**Pull request by christiezhao-outreach**
## Jira ID

[XX-XX]

### CODE REVIEW

File: manifest.yaml [hunk 1]
```diff
@@ -138,39 +138,27 @@ arguments:
...
     schema:
       type: number
-    description: Overrides the high traffic threshold in the "grpc_latency_high" terraform module in monitoring/datadog.tf.
+    description: Overrides the high traffic threshold in the "grpc_success_rate_low" terraform module in monitoring/datadog.tf.
   terraform.datadog.http.evaluationWindow:
     schema:
       type: string
     description: The window of time to evaluate metrics related to HTTP events for each monitor and alert.
-  terraform.datadog.http.percentiles.lowTraffic:
+  terraform.datadog.http.qos.percentiles.lowTraffic:
```
Comment (jkinkead) [inline manifest.yaml:150]: These renames don't look backwards-compatible. I know we've added backcompat support before for things; maybe @jaredallard can chime in when he's back.

**Review by jkinkead — COMMENTED** [review]
Thanks for the contribution! Overall this looks pretty good, although I'm a little worried about the breaking changes here.

From what I can tell, the main change is adding a configurable time window?

File: manifest.yaml (hunk 1 above)
Comment (christiezhao-outreach) [reply manifest.yaml:150]: The main change is change the newly added http.count.highcount to lowcount and add the time window. Generally with the http latency variables added, in `service.yaml` it wasn't as clear as gRPC setup which has the success rate and latency high differentiated very clearly. But we can close this ticket if this is too big of a change for everybody!

Comment (artemchernyak-outreach) [reply manifest.yaml:150]: I like the addition of the time window to the metrics. But I also agree with Jesse on the rename being unnecessary.

Here are a couple options to make this mergeable:

- Remove the renaming changes, but keep the window changes.
- Keep the old names and add the new names along side. Then you can do `{{ oldName | newName }}` in the template.

We really don't want to change these stencil values unexpectedly especially now that we don't have an overarching running (like bootstrap) that can automatically change these for teams.

Comment (jkinkead) [reply manifest.yaml:150]: +1 to what Artem said. The new configurable value is great, and the renames are making the HTTP variables match the gRPC variables, if I read that correctly. Also great! But you should do one of the actions Artem suggested.

**Comment by jaredallard** [comment]
Closing this, if you still feel like you wish to get this out @christiezhao-outreach, please feel free to take it on :D

**Pull request closed without merging**